src/core/script/api/internals.js

```diff
@@ -0,0 +1,31 @@
+import state from './state.js';
+import engine from './engine.js';
+import options from './options/options.js';
+import dom from './utilities/dom/index';
+import ns from './utilities/namespace.js';
+import support from './utilities/support/index';
+import property from './utilities/property/index';
+
+const internals = [];
```

Comment:
pourquoi un array et pas un objet ?
```suggestion
const internals = {};
```

Reply:
yes c'est vrai ça, pourquoi ??
je corrige merci